app.js

```diff
@@ -3,16 +3,21 @@ module.exports = (/*options*/) => {
   const express = require('express');
   const app = express();
   const bodyParser = require('body-parser');
-  const config = require('./config.json');
-  const request = require('request');
+  const https = require('https');
+  const axios = require('axios');
   const crypto = require('crypto');
   const fs = require('fs');
+  const quayHost = process.env.QUAY_HOST;
```

Comment:
Do we need to validate and throw with a message that this is required?

Reply:
Yeah, we probably do.